src/app/cli/cli_open_file.h

```diff
@@ -37,6 +37,7 @@ namespace app {
     bool listSlices;
     bool ignoreEmpty;
     bool trim;
+    bool trimByGrid;
```

Comment:
About this field you have two options:

1. Remove it from this PR (because it's not used)
2. Or complete this PR with the trim-by-grid support for `--save-as`